site/layouts/partials/title.html

```diff
@@ -1,11 +1,14 @@
+{{ if isset .Params "course_id" }}
```

Comment:
Are there any pages which don't already have a `course_id`?

Reply:
Boilerplate pages like the home page, search page and course index don't, and they still call this partial because it's called by the `head.html` partial which is called on every page because it's called in `site/layouts/_default/baseof.html`.